packages/workflow/src/utils.ts

```diff
@@ -8,6 +10,7 @@ export const deepCopy = <T>(source: T, hash = new WeakMap(), path = ''): T => {
 		return source;
 	}
 	if (hash.has(source)) {
+		ErrorReporterProxy.getInstance().warn(`Circular reference detected at "source${path}"`);
```

Comment:
At a glance, `LoggerProxy` does not seem to need `getInstance()` to be used. Is it possible to have `ErrorReporter.error(error)` without `getInstance()`?